### PR TITLE
Add the logo, fix grammar, and small tweaks.

### PR DESCRIPTION
**Pull request by HelenaMKG**
Please accept this, I spent an embarrassing amount of time figuring out alignment.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,20 @@
-# Blaze 4D **Early Development**
+![blaze](https://user-images.githubusercontent.com/68126718/125143247-71be4580-e0f0-11eb-88bc-070eb2838435.png)
 
 ## Information 
-Blaze4D is a fabric mod that changes Minecraft's rendering engine to use the Vulkan Graphics Library.\
-Blaze4D is currently in Early Development and should **NOT** be used if you do **NOT** have somewhat experience in building Fabric mods.\
+Blaze4D is a Fabric mod that changes Minecraft's rendering engine to use the Vulkan Graphics Library, it is currently in Early Development and is **NOT** intended for use by the general public.\
 Support for Blaze4D **WILL NOT** be given as the mod is in its **Early Development** stages.\
-Blaze4D is compatible with Sodium.
```
Comment (halotroop2288) [inline README.md:7]: Don't remove this line. It's factually true, not just hydos' opinion.

Comment (HelenaMKG) [reply README.md:7]: Okay, sorry, I agree with you.

**Review by hYdos — APPROVED** [review]
im not a huge fan on the "A vulkanized minecraft" part but it works for me

**Review by halotroop2288 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ramidzkh) [reply README.md:7]: Depends on how far you want to spread out 'it works'

Comment (halotroop2288) [reply README.md:7]: I'd say just about as long as combining the two doesn't cause issues that either one by itself does, they "work" together. (You can't expect much from Blaze4D at this stage, but as long as adding Sodium doesn't break it *more*, it's good.
And hydos has done his best to ensure a minimum amount of compatibility between the two up to this point. (I.E. Making sure it doesn't crash the game.)

**Pull request merged**